trackersite/tracker/templates/tracker/ticket_table.html

```diff
@@ -1,33 +1,99 @@
 {% load i18n trackertags %}
-<table class="ticket_list">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
```

Comment:
Do static souborů je přidaná vlastní kopie, asi bude lepší použít tu.

Reply:
Aneb když to člověk stahuje a zapomene změnit v kódu...